.cirrus.yml

```
@@ -59,7 +59,7 @@ task:
     - ./ci/lint_run_all.sh
 
 task:
-  name: 'ARM [unit tests, no functional tests] [buster]'
+  name: 'ARM [unit tests, no functional tests] [bullseye]'
   << : *GLOBAL_TASK_TEMPLATE
```

YAMLlint (1.35.1)

[warning] too many spaces before colon

(colons)